jstz_cli/src/run.rs

```diff
@@ -47,6 +48,7 @@ pub async fn exec(
             method,
             headers: HeaderMap::default(),
             body,
+            gas_limit: gas_limit.try_into().unwrap_or(usize::MAX),
```

Comment:
Why not use `usize`?

Reply:
On most architectures `usize` is a `u64`, but json doesn't handle `u64`s correctly so we have to pass a `u32`.
The only reason `|x : u32| -> usize x.try_into().unwrap()` would fail is on an architecture with `usize` represented as a `u16` (ie never) 
There could also be an issue as `usize` is most likely `u64` in the cli but a `u32` in wasm32. This will change with risc-v though.